web/frontend/src/pages/form/components/utils/useChangeAction.tsx

```diff
@@ -314,35 +314,64 @@ const useChangeAction = (
 
   useEffect(() => {
     if (userConfirmedAddVoters.length > 0) {
-      const newUsersArray = [];
-      for (const sciperStr of userConfirmedAddVoters.split('\n')) {
+      let sciperErrs = '';
+
+      const providedScipers = userConfirmedAddVoters.split('\n');
+      setUserConfirmedAddVoters('');
+
+      for (const sciperStr of providedScipers) {
+        const sciper = parseInt(sciperStr, 10);
+        if (isNaN(sciper)) {
+          sciperErrs += t('sciperNaN', { sciperStr: sciperStr });
+        }
+        if (sciper < 100000 || sciper > 999999) {
+          sciperErrs += t('sciperOutOfRange', { sciper: sciper });
+        }
```

Comment:
This should use `authManager.readSCIPER`

Reply:
but that's defined in the backend

Reply:
Oups...